### ISSUE TITLE
Google Map URL Format

### ISSUE THREAD

**Issue by anmamtl** [jsor/lity#122]
Hello!

I've been trying to understand what format I had to have for the google map to show.
I tried many urls types, all from the google map site (embed, url, short link...) but it doesn't seem to work.
Could you point me to the right direction?

Thank you ;)

**Comment by jsor** [comment]
Yes, Lity supports the kinda old format, see https://github.com/jsor/lity/blob/master/test/functional/uri.html#L122

I need to update the Google Maps URL handling to also support the current structure.

**Comment by elisafern12** [comment]
Hey there ;)

Any updates on this? Im never able to find the right URL to put :/

Thanks a lot!

**Comment by jsor** [comment]
I'm not able to work on this at the moment. But it's still on the todo list.

**Comment by elisafern12** [comment]
Ok, I understand - but do you know how I can generate the old URL somewhere or what is the structure?

Thanks!

**Comment by bitpunk** [comment]
Go to Google Maps and search your location. Check the current URL format: 

[…]/maps/place/**1600+Amphitheatre+Pkwy,+Mountain+View,+CA+94043,+USA**/@37.4216946,-122.0854278,17z/[…]

Simply copy that bold part and create your lity URL like that:

[…]/maps?q=**1600+Amphitheatre+Pkwy,+Mountain+View,+CA+94043,+USA**
